src/styles/form.css

```diff
@@ -13,6 +13,11 @@
   text-transform: none;
 }
 
+/* Focus */
+.ui.form .field:focus-within > label {
+  color: var(--primary-pms-362-900);
```

Comment:
This will color the label green on focus.

Reply:
Thank you!!